lib/apiForSheets.js

```diff
@@ -4,15 +4,13 @@ const readline = require('readline');
 const google = require('googleapis');
 const googleAuth = require('google-auth-library');
 const claLabel = ['PR: don\'t merge - NEEDS CLA'];
```

Comment:
Why is this an array?

Reply:
This is an array because when adding labels on PRs, we need to send in an array of labels here: https://github.com/oppia/oppiabot/pull/24/files#diff-37ff2aeafde1de238e19c818c2265b4aR95
Therefore, the labels are stored as string arrays.